ci-operator/jobs/openshift/cloud-provider-gcp/openshift-cloud-provider-gcp-main-presubmits.yaml

```diff
@@ -586,3 +586,59 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )verify-commits,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^main$
+    - ^main-
+    cluster: build09
+    context: ci/prow/verify-vendor
+    decorate: true
+    labels:
+      ci.openshift.io/generator: prowgen
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-cloud-provider-gcp-main-verify-vendor
+    optional: true
+    path_alias: k8s.io/cloud-provider-gcp
+    rerun_command: /test verify-vendor
```

Comment:
You could also add this line, taken from another job in this file

```
skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
```

This is small, but it will avoid running this job if no code was changed.

Reply:
Thanks, done!